polkadot-protocol/architecture/polkadot-chain/pos-consensus.md

```diff
@@ -111,13 +101,29 @@ Key features of BEEFY include:
 - **ECDSA signature schemes** - BEEFY uses ECDSA signatures, which are widely supported on Ethereum and other EVM-based chains, making integration with these ecosystems smoother
 - **Light client optimization** - BEEFY reduces the computational burden on light clients by allowing them to check for a super-majority of validator votes rather than needing to process all validator signatures, improving performance
 
-??? interface "Additional Information"
+## Additional Resources
```

Comment:
It sort of feels like the resources should be where they were previously. It seems a bit weird to have a BABE section above and then a separate section under Additional Resources related to BABE. Just looking at the TOC makes me question this a little bit